src/optionParser.ts

```diff
@@ -36,6 +37,7 @@ commander
   .option("--user-agent <string>", "user agent", "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3312.0 Safari/537.36")
   .option("--timeout <number>", "time out ms", 30000)
   .option("--wait-for <number>", "wait for ms", 250)
+  .option("--landscape <number>", "landscape pdf file", false)
```

Comment:
landscape option must be `boolean`.
https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagepdfoptions

```suggestion
  .option("--landscape", "landscape pdf file", false)
```